src/Arcus.Messaging.Pumps.ServiceBus/Configuration/AzureServiceBusMessagePumpOptions.cs

```diff
@@ -10,6 +10,7 @@ public class AzureServiceBusMessagePumpOptions
     {
         private int? _maxConcurrentCalls;
         private string _jobId;
+        private TimeSpan _keyRotationTimeout = TimeSpan.FromSeconds(5);
```

Comment:
Let's open it up for those who want to, but use a default

Reply:
Yes, did that. This is only the default. It is added in the `AzureServiceBusPumpConfiguration` base class so both topic and queue registrations can benifit from this.
This marked line is only to have an internal default.

Reply:
Oh nevermind, looked over the other file!